fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/NopTarget.java

```diff
@@ -12,46 +12,39 @@
 import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
 
 /**
- * An {@link IDatabaseTarget} which just acts as a sink and doesn't actually
- * do anything. Useful for tests, just to exercise the code.
+ * An {@link IDatabaseTarget} which just acts as a sink and doesn't actually do
+ * anything. Useful for tests, just to exercise the code.
+ * 
  * @author rarnold
  *
  */
 public class NopTarget implements IDatabaseTarget {
-    
+
     /**
      * Public constructor
-     * @param decorated
      */
     public NopTarget() {
+        // No operation
     }
-    
-    /* (non-Javadoc)
-     * @see com.ibm.fhir.database.utils.api.IDatabaseTarget#runStatement(java.lang.String)
-     */
```

Comment:
happy to see these go.  our use of these is extremely inconsistent and i'm not sure that they add much value.  i deleted a bunch in my last PR as well